src/components/home/section-dataset-cloud/section-dataset-cloud.tsx

```diff
@@ -9,21 +9,21 @@
 import React, { FC } from "react";
 
 // App dependencies
+import { DatasetCloudStaticQuery } from "../../../hooks/dataset-cloud-query";
```

Comment:
Again, `prettier` reordered the imports. The change here is the commenting out of the CTA button `Consortia Roadmap` until the project is live!